packages/shared/components/enums/logo.enum.ts

```diff
@@ -0,0 +1,6 @@
+export enum Logo {
+    LogoFireflyFull = 'logo-firefly-full',
+    LogoFirefly = 'logo-firefly',
+    LogoStronghold = 'logo-stronghold',
+    LogoChrysalisGem = 'logo-chrysalis-gem',
+}
```

Comment:
Perhaps the enum name could remove the `Logo` prefix because its already in the logo enum?

```suggestion
export enum Logo {
    FireflyFull = 'logo-firefly-full',
    Firefly = 'logo-firefly',
    Stronghold = 'logo-stronghold',
    ChrysalisGem = 'logo-chrysalis-gem',
}
```